### PR TITLE
Change the vocab for "platform" GA

### PR DESCRIPTION
**Pull request by vturpin**
Following issue #71 suggestion
Change "Autonomous Underwater Vehicle" into "sub-surface gliders". and link to the proper vocab

Proponents: 
Moderator: @OceanGlidersCommunity/format-mantainers

# Type of PR

- [ ] Typo without possible change of interpretation of the related text.
- [ ] Fix of some error, inconsistency, unforeseen limitation.
- [ ] Style that only affects visually the compiled document
- [X] Addition that does not require change in the current structure.
- [ ] Enhancement that require changes to improve the format.

# Related Issues

# Dates when it got review approvals

#71 pushed june 2022.

# Release checklist

- [x] Approved by at least two members of the committee?
- [ ] There were modifications after the review approvals? If so, please
      ask reviewers to update their review.
- [x] Proponents and moderador should explicitly agree that it is ready to
      to merge.
- [x] The moderador is the one in charge to actually merge or close this PR
      according to the final decision.

# For maintainers

- [ ] Update the moderator with a volunteer from the committee. It would be
      best to have one single moderator to guide and help this PR to move
      forward. It is OK to update the moderador pass it to another one.
- [ ] Confirm that the associated branch was deleted after the merging.
- [ ] Wrap-up and close the related issues.

# Comments

### CODE REVIEW

File: OG_Format.adoc [hunk 1]
```diff
@@ -133,8 +133,8 @@ Name of the platform(s) that supported the sensors data used to create this data
 
 https://vocab.nerc.ac.uk/collection/L06/current/[_https://vocab.nerc.ac.uk/collection/L06/current/_]
 
- |mandatory |“Autonomous Underwater Vehicle”
-|platform_vocabulary |Controlled vocabulary for the names used in the "platform" attribute. |mandatory |https://vocab.nerc.ac.uk/collection/L06/current/27/[_https://vocab.nerc.ac.uk/collection/L06/current/27/_]
+ |mandatory |“”
```
Comment (castelao) [inline OG_Format.adoc:136]: Isn't missing 'sub-surface gliders' ?

**Review by castelao — CHANGES_REQUESTED** [review]

File: OG_Format.adoc (hunk 1 above)
Comment (vturpin) [reply OG_Format.adoc:136]: Yes. 
Error in the change. I will deleate and re-do the PR.

Comment (castelao) [reply OG_Format.adoc]: I just edited it for you.

File: OG_Format.adoc [hunk 2]
```diff
@@ -133,8 +133,8 @@ Name of the platform(s) that supported the sensors data used to create this data
 
 https://vocab.nerc.ac.uk/collection/L06/current/[_https://vocab.nerc.ac.uk/collection/L06/current/_]
 
- |mandatory |“Autonomous Underwater Vehicle”
-|platform_vocabulary |Controlled vocabulary for the names used in the "platform" attribute. |mandatory |https://vocab.nerc.ac.uk/collection/L06/current/27/[_https://vocab.nerc.ac.uk/collection/L06/current/27/_]
+ |mandatory |"sub-surface gliders"
+|platform_vocabulary |Controlled vocabulary for the names used in the "platform" attribute. |mandatory |https://vocab.nerc.ac.uk/collection/L06/current/27/[_https://vocab.nerc.ac.uk/collection/L06/current/25/_]
```
Comment (castelao) [inline OG_Format.adoc:137]: I just noticed an inconsistency in the text vs link. Is it 27 or 25?

**Review by castelao — CHANGES_REQUESTED** [review]

File: OG_Format.adoc (hunk 2 above)
Comment (vturpin) [reply OG_Format.adoc:137]: Hum, nice. Thanks.
Yes this is 27... le right vocab is "sub-surface gliders".

**Review by callumrollo — APPROVED** [review]

**Review by JuangaSocib — APPROVED** [review]

**Comment by vturpin** [comment]
@castelao I dismiss your previous "request change" on this. After being implemented. I thought that was the right thing to do. But not. However, there is no impact. I change the uri to the right one. I still need your review on this to merge.

**Review by castelao — APPROVED** [review]

**Review by justinbuck — COMMENTED** [review]
Approved

**Pull request merged**